lib/canvas.js

```diff
@@ -44,6 +44,25 @@ Canvas.prototype.createJPEGStream = function (options) {
   return new JPEGStream(this, options)
 }
 
+/**
+ * The OffscreenCanvas.convertToBlob() method creates a Blob object representing
+ * the image contained in the canvas.
+ *
+ * @see https://developer.mozilla.org/en-US/docs/Web/API/OffscreenCanvas/convertToBlob
+ * @since NodeJS v18.0.0
+ */
+Canvas.prototype.convertToBlob = async function (options = {}) {
+  const type = options.type && FORMATS.includes(options.type)
```

Comment:
This seemed quite stupid, so I think it's best to include a link that this is done according to spec 😅 

```suggestion
  // If the user agent does not support the requested type, then it must create the file using the PNG format.
  // ref: https://html.spec.whatwg.org/multipage/canvas.html#a-serialisation-of-the-bitmap-as-a-file
  const type = options.type && FORMATS.includes(options.type)
```